conf/modules.config

```
@@ -245,4 +245,13 @@ process {
         ]
     }
 
+    withName: KAIJU_KAIJU {
+        publishDir = [
+            path: { "${params.outdir}/kaiju/${meta.db_name}" },
+            mode: params.publish_dir_mode,
+            pattern: '*.tsv'
+        ]
+        ext.args = { "${meta.db_params}" }
+        ext.prefix = { "${meta.id}-${meta.run_accession}-${meta.db_name}" }
```

```suggestion
        ext.prefix = params.perform_runmerging ? { "${meta.id}-${meta.db_name}" } : { "${meta.id}-${meta.run_accession}-${meta.db_name}" }
```

It would also be good if you could move this up to the other profilers

---

I added this to centrifuge profiler as well.